### ISSUE TITLE
Importing `GlobalStyle` and a theme doesn't work in Gatsby environments.

### ISSUE THREAD

**Issue by liyun-li** [React95/React95#446]
Putting the following lines in either `gatsby-browser.ts` or directly in a page of Gatsby has no effect on the styling, and I do have `styled-component` installed. I checked out `React95/gatsby-theme` but decided to just use `@react95/core` because I don't use features like `mdx` etc.

```js
import '@react95/core/GlobalStyle';
import '@react95/core/themes/win95.css';
```

**Comment by izwb003** [comment]
May try doing this? I saw it in [/core/.storybook/decorators/Frame.tsx](https://github.com/React95/React95/blob/53753414ed682eb7ac014b72339412472487600e/packages/core/.storybook/decorators/Frame.tsx) which fixes my missing style issue #445 :
```js
import * as GlobalStyle from '@react95/core/GlobalStyle';
import * as themes from '@react95/core/themes/win95.css';

console.log({ GlobalStyle }, { themes });
```

The ```console.log``` is necessary, which is pretty strange... I guess the original import syntax would cause these two import sentences being optimized and automatically deleted, as they were not explicitly referenced in the following text - WebStorm even directly grayed out these two sentences. Then by using them once with ```console.log()``` the system will consider they are useful so they will not be optimized and works.

**Comment by ggdaltoso** [comment]
Hi, @liyun-li, and thanks for opening this issue.

I was trying to understand this issue and I think I got it fixed in v8.0.1 we just shipped. Could you, please, try it and double-check?

@izwb003 thanks for your suggestion but I think the issue was bigger. Tried to explain it a little more in this PR description #449.
